barretenberg/cpp/src/barretenberg/goblin/mock_circuits.hpp

```diff
@@ -136,41 +136,6 @@ class GoblinMockCircuits {
         MockCircuits::construct_arithmetic_circuit(builder);
     }
 
-    /**
-     * @brief Construct a size 2^17 mock kernel circuit based on vanilla recursion for benchmarking
-     * @details This circuit contains (1) some arbitrary operations representing general kernel logic, (2) recursive
-     * verification of a function circuit proof, and optionally (3) recursive verification of a previous kernel circuit
-     * proof. The arbitrary kernel logic is structured to bring the final dyadic circuit size of the kernel to 2^17.
-     *
-     * TODO(https://github.com/AztecProtocol/barretenberg/issues/801): Pairing point aggregation not implemented
-     * @param builder
-     * @param function_accum {proof, vkey} for function circuit to be recursively verified
-     * @param prev_kernel_accum {proof, vkey} for previous kernel circuit to be recursively verified
-     */
-    static void construct_mock_recursion_kernel_circuit(MegaBuilder& builder,
```

Comment:
why delete?

Reply:
This functionality is not and never will be needed